health-services/facility/src/main/java/org/egov/facility/web/controllers/FacilityApiController.java

```diff
@@ -111,15 +112,17 @@ public ResponseEntity<FacilityBulkResponse> facilityV1SearchPost(
             @Valid @ModelAttribute URLParams urlParams,
             @ApiParam(value = "Details for existing facility.", required = true) @Valid @RequestBody FacilitySearchRequest request
     ) throws Exception {
-        List<Facility> facilities = facilityService.search(
+        SearchResponse<Facility> searchResponse = facilityService.search(
                 request,
                 urlParams.getLimit(),
                 urlParams.getOffset(),
                 urlParams.getTenantId(),
                 urlParams.getLastChangedSince(),
                 urlParams.getIncludeDeleted());
         FacilityBulkResponse response = FacilityBulkResponse.builder().responseInfo(ResponseInfoFactory
-                .createResponseInfo(request.getRequestInfo(), true)).facilities(facilities).build();
+                .createResponseInfo(request.getRequestInfo(), true))
+                .facilities(searchResponse.getResponse())
+                .totalCount(searchResponse.getTotalCount()).build();
```

Comment:
Total count will be set 0 for ids search, it should be facility.size()

Reply:
Done